notifications.bs

```diff
@@ -189,6 +212,24 @@ or vibration pattern that is not otherwise accessible to the end user.
   <li><p>If <var>options</var>'s <code>sticky</code> is true, set
   <var>notification</var>'s <a>sticky preference flag</a>.
 
+  <li><p>Set <var>notification</var>'s list of <a>actions</a> to an empty list,
```

Comment:
Hmm, the prevailing style seems to be not to indent these - see [line 289][1],  [line 306][2],  [line 351][3],  [line 369][4], and  [line 413][5].

[1]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L289
[2]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L306
[3]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L351
[4]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L369
[5]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L413

The only examples I can find of indenting the children - [line 434][6],  [line 581][7], and  [line 594][8] - all use single-space indent for the entire enclosing `<ol>`. But I'd rather not convert other `<ol>`s like the giant [create-a-notification](https://notifications.spec.whatwg.org/#create-a-notification) `<ol>` to single-space indent.

[6]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L434
[7]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L581
[8]: https://github.com/whatwg/notifications/blob/f72a0417b9cacc3e29194bc7e0e602d97224e5a5/notifications.bs#L594

Do still you want me to indent these? And if so, shall I use 2-space indent even though there's no precedent for that?